src/__tests__/safeStringify.ts

```diff
@@ -44,6 +44,12 @@ describe('safeStringify', () => {
   it('should work', () => {
     const val = { foo: true };
     expect(safeStringify(val)).toEqual('{"foo":true}');
-    expect(safeStringify('{"foo":true}')).toEqual('{"foo":true}');
+    expect(safeStringify('{"foo":true}')).toEqual(JSON.stringify('{"foo":true}'));
```

Comment:
:+1:

Reply:
Did this cause a bug somewhere? This is actually intentional, and a pretty big change that will likely break all sorts of random places in next. We deal with so much user driven input that often objects are temporarily strings (such as when user removes a comma in json and its temporarily invalid and thus turns into a string. We don't want to re-stringify that object and add all the escapes like this would do. If we put this in, we need to add defensive checks in many places that use safeStringify to not call it if the input is already a string.